hostscripts/gatehost/freshadminvm

```diff
@@ -25,13 +25,15 @@ common_scripts="mkcloud-onhost.sh mkcloud-common.sh mkcloud-driver-$mkclouddrive
 for script in $common_scripts; do
     source ${scripts_lib_dir}/$script
 done
-admin_node_disk=/dev/system/$n
-[ -e $admin_node_disk ] || admin_node_disk=/dev/gate/$n
-[ -e $admin_node_disk ] || exit 51
+if [[ !-z $admin_node_disk ]]; then
```

Comment:
the '!' should not be here - it is a syntax error and we do not want to negate here anyway.

Reply:
well spotted ;-) fixing.